src/libecalc/testing/yaml_builder.py

```diff
@@ -115,32 +111,6 @@ def with_consumption_rate_type(self, consumption_rate_type: ConsumptionRateType)
 TYamlClass = TypeVar("TYamlClass", bound=YamlBase)
 
 
-class YamlModelContainer(Generic[TYamlClass]):
-    """
-    Consumer, Installation wrapper that can be used to bundle the component with its dependencies.
-
-    A fuel consumer that uses a model reference can then use this class to keep the referenced model in a single class.
-    """
-
-    def __init__(self, component: TYamlClass):
-        self.component = component
-        self.models = None
-        self.time_series = None
-        self.facility_inputs = None
-
-    def with_models(self, models) -> Self:
-        self.models = models
-        return self
-
-    def with_time_series(self, time_series):
-        self.time_series = time_series
-        return self
-
-    def with_facility_inputs(self, facility_inputs) -> Self:
-        self.facility_inputs = facility_inputs
-        return self
-
-
```

Comment:
Not used (yet).

Had to make some changes in yaml_builder since mypy now checks it. I guess we don't run mypy on tests..